src/utils/displayResult.js

```diff
@@ -2,25 +2,39 @@ import chalk from 'chalk'
 
 export function displayResult(err, failureMessage, successMessage) {
```

Comment:
It should be pretty straightforward to add unit tests for this function to test all the scenarios.